frontend/src/components/world/WorldMap.tsx

```diff
@@ -424,6 +426,20 @@ class CoveyGameScene extends Phaser.Scene {
     this.input.keyboard.addCapture(this.previouslyCapturedKeys);
     this.previouslyCapturedKeys = [];
   }
+
+  teleport(intoMaze: boolean) {
+    if(this.player) {
+      const body = this.player.sprite.body as Phaser.Physics.Arcade.Body;
```

Comment:
oh this is pretty interesting, did it not make sense to just make an update location with fixed coordinates?